ServerSideDemo/EnvVariableBlazorGoogleMapsKeyService.cs

```diff
@@ -13,7 +13,7 @@ public Task<MapApiLoadOptions> GetApiOptions()
         var key = Environment.GetEnvironmentVariable("GOOGLE_MAPS_API_KEY");
         if (string.IsNullOrEmpty(key))
         {
-            key = "AIzaSyBdkgvniMdyFPAcTlcZivr8f30iU-kn1T0";
+            key = "AIzaSyCmIwKIsEjIxZ6EUecb2paIIah8defiDig";
```

Comment:
Please revert back. I suggest to set your key to GOOGLE_MAPS_API_KEY env variable